src/utils/HtmlUtils.ts

```diff
@@ -104,14 +104,10 @@ export class ImageUtils {
     options = options ? options : <IImageUtilsOptions>{};
 
     let img = ImageUtils.buildImage(undefined, _.extend(options, { 'data-coveo-uri-hash': result.raw['urihash'] }));
-    if (endpoint.isJsonp()) {
-      // For jsonp we can't GET/POST for binary data. We are limited to only setting the src attribute directly on the img.
-      ImageUtils.buildImageWithDirectSrcAttribute(endpoint, result);
-    } else {
-      // Base 64 img allows us to GET/POST the image as raw binary, so that we can also pass the credential of the user
-      // Useful for phonegap.
-      ImageUtils.buildImageWithBase64SrcAttribute(endpoint, result);
-    }
+    // Base 64 img allows us to GET/POST the image as raw binary, so that we can also pass the credential of the user
+    // Useful for phonegap.
```

Comment:
Phonegap 😂  ... Oh boy, bad memories.

Reply:
I barely used it and I still remember the performance of it.